src/_shared/js/events-multiple.js

```diff
@@ -59,18 +62,18 @@ function importCard(adType) {
   }
 }
 
-const boldTitle = (title) => {
-    return title.split(":")[1] ? `<b>${title.split(":")[0]}:</b>${title.split(":")[1]}` : `<b>${title}</b>`
-}
+const boldTitle = title => {
+  return title.split(":")[1]
```

Comment:
what happens if theres no `title` or a `title` without a semicolon?

Reply:
Title should always exist. The Scala endpoint has it as `String` rather than `Option[String]`